tests/EventParser.agent.test.nut

```diff
@@ -39,7 +39,7 @@ class EventParserTestCase extends ImpTestCase {
         events = _firebase._parseEventMessage("{\"path\":\"/\",\"data\":{\"data\":214}}\n");
         assertEqual(1, events.len());
         assertEqual(214, events[0].data.data);
-        _firebase._bufferedInput = "";
+        assertEqual(_firebase._bufferedInput, "");
```

Comment:
It should be the other way around:

`assertEqual("", _firebase._bufferedInput);` - expected value goes first.